src/ssl.c

```diff
@@ -26335,35 +26382,80 @@ WOLFSSL_API int wolfSSL_CTX_set1_curves_list(WOLFSSL_CTX* ctx, char* names)
 #endif
 
 #ifdef OPENSSL_EXTRA
+#ifndef NO_WOLFSSL_STUB
 int wolfSSL_CTX_set_msg_callback(WOLFSSL_CTX *ctx, SSL_Msg_Cb cb)
 {
     WOLFSSL_STUB("SSL_CTX_set_msg_callback");
     (void)ctx;
     (void)cb;
     return WOLFSSL_FAILURE;
 }
+#endif
+
 int wolfSSL_set_msg_callback(WOLFSSL *ssl, SSL_Msg_Cb cb)
```

Comment:
Can you add a unit test for wolfSSL_set_msg_callback()?  Have you verified that this works as expected?

Reply:
Yes, in test_wolfSSL_set_options, please find SSL_set_msg_callback.